packit_service/models.py

```diff
@@ -176,6 +176,33 @@ def get_projects(cls, first: int, last: int) -> Iterable["GitProjectModel"]:
             )[first:last]
             return projects
 
+    @classmethod
+    def get_namespace(cls, forge: str, namespace: str) -> Iterable["GitProjectModel"]:
+        """Return projects of given forge and namespace"""
+        with get_sa_session() as session:
+            projects = (
+                session.query(GitProjectModel).filter_by(namespace=namespace).all()
+            )
+            matched_projects = []
+            for project in projects:
+                forge_domain = urlparse(project.project_url).netloc
+                if forge == forge_domain:
+                    matched_projects.append(project)
```

Comment:
Are you sure this would be the right usage for that function? What would be the additional benefit? It's only the domain name that's needed here, `parse_git_repo` returns many other things which would not be used here.

Reply:
`urlparse` needs to parse more things as well.

Benefits?

- Reusing our own code. Having the same way to get this info across the code-base. (We already use it in `packit`/`packit-service`.)
- I was confused about the `netloc` name.
- Dogfooding.

Reply:
> * Reusing our own code. Having the same way to get this info across the code-base. (We already use it in `packit`/`packit-service`.)

That code would do the same thing as this code. Again: what's the benefit?

> * I've been confused about the `netloc` name. =>  `hostname` looks more clear to me

And you were rightly so: we should probably use `hostname` here, instead of `netloc`. Nice catch! :rocket: 

> * Dogfooding.

I would say that calling a function to call a library function is and extra step, which should be removed :slightly_smiling_face:

Reply:
> That code would do the same thing as this code. Again: what's the benefit?

It's nearly the same, so we can choose both. I already wrote three benefits. We have multiple occurrences of the URL parsing -- some use more and some less of that function/class. My suggestion was to use the same function for all to be consistent.

That was meant as a suggestion of alternative approach. I don't care what we choose. My main problem was with the property name.